setup.py

```diff
@@ -28,7 +28,7 @@
         "jsonschema>=3.2.0",
         "marshmallow>=3.0.0",
         "microcosm>=4.0.0",
-        "microcosm-logging>=2.0.0",
+        "microcosm-logging>=2.1.0",
```

Comment:
once [this](https://github.com/globality-corp/microcosm-logging/pull/65) is merged and released this version will be available - should fix remaining test-failures in CI for this PR cc/ @salmanmashayekh @lfmoisa-globality